build/Targets/Packages.props

```diff
@@ -376,6 +375,7 @@
       https://vside.myget.org/F/vs-impl/api/v3/index.json;
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://dotnet.myget.org/F/system-commandline/api/v3/index.json;
+      \\cpvsbuild\drops\VS\d16.0\raw\28222.01\binaries.x86ret\archives\vssdk\signed\ENU\x86;
```

Comment:
This obviously won't be merged.

Reply:
Did you not run into the graphmodel problem that I did?

Reply:
I did, you'll see the build warnings in our build.